README.md

```diff
@@ -24,7 +24,10 @@ source ./venv/bin/activate
 pip3 install -r requirements.txt 
 ```
 
-- Please download Firebase Admin SDK service json file from Firebase Console and put that in project's root directory. Rename file to firebase_cred.json
+- Please download Firebase Admin SDK service json file from Firebase Console, You can download the file by following below steps:
+    1. Go to `Project Settings/Service accounts`
+    1. Click on `Generate New Primary Key`
```

Comment:
@raghavjoshi789 Rectify the list numbering.

Reply:
@Rahulm2310  Should I remove points or only formatting